components/separator/src/tokens.json

```diff
@@ -25,7 +27,7 @@
             "syntax": "<length>",
             "inherits": true
           },
-          "nl.nldesignsystem.figma.supports-token": false
+          "nl.nldesignsystem.figma.supports-token": true
```

Comment:
@jeffreylauwers Ik dacht dat margin tokens niet supported waren, klopt het dat deze `true` zijn?

Reply:
<img width="725" alt="Screenshot 2024-08-31 at 15 50 21" src="https://github.com/user-attachments/assets/bb8d354f-9cff-4755-96dd-6130b9f28eeb">

<img width="988" alt="Screenshot 2024-08-31 at 15 50 13" src="https://github.com/user-attachments/assets/8284ae68-dfe3-498f-b3c1-66155ec06d08">

@jeffreylauwers Het is in Figma geïmplementeerd als padding zie ik, dat lijkt me niet ideaal, want dan krijg je toch teveel witruimte als je ook nog een gap zou gebruiken met autolayout?

Reply:
Yes, precies waar we het vrijdag bij de Design Open dag over gehad hebben. Ik ga dat eruit slopen en de tokens op figma support false zetten.